awx/main/migrations/0079_v360_rm_implicit_oauth2_apps.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-28 17:15
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0078_v360_clear_sessions_tokens_jt'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='oauth2application',
+            name='authorization_grant_type',
+            field=models.CharField(choices=[('authorization-code', 'Authorization code'), ('password', 'Resource owner password-based')], help_text='The Grant type the user must use for acquire tokens for this application.', max_length=32),
```

Comment:
What happens on upgrade if an existing `OAuth2Application` has `authorization_grant_type="implicit"` ?

Reply:
@ryanpetrello Existing implicit apps will continue to exist.  We don't currently purge them from the db.  I verified that those apps can still be used to create tokens using the implicit flow, this PR will just make it so that users can't create _new_ apps with the `implicit` grant type.

Reply:
So `choices` here doesn't actually _enforce_ the value at the database level?

Reply:
nope.  I did the following:
1. checked out devel
2. created an implicit application
3. checked out this branch and migrated
4. observed that the implicit application still exists & that I can still make a token using the implicit flow 

```
https://localhost:8043/api/o/authorize/?response_type=token&client_id=BDTpHIV9QkHzT03mrhXeoL8uHX6HLeQBgRKgvKA4&scope=read
```